.circleci/config.yml

```diff
@@ -195,7 +195,7 @@ jobs:
           command: pytest tests/tasks -vvrfEsx --numprocesses 4 --dist=loadfile
 
   # ----------------------------------
-  # Run unit tests in Python 3.6-3.8
+  # Run unit tests in Python 3.6-3.9
```

Comment:
```suggestion
  # Run unit tests in Python 3.7-3.9
```

Reply:
We still test 3.6 on Unix; I just did not include it in the Windows tests

Reply:
I see, makes sense